languages/rust/exercises/concept/structs/.docs/after.md

```diff
@@ -0,0 +1,7 @@
+Notice the usage of `&self` (an _immutable_ reference to our `User` instance) for the `get_` methods vs. the usage of `&mut self` (a _mutable_ reference to our `User` instance) for the `update_` methods. The `get_` methods don't need a mutable reference since their job is to fetch some data; they aren't looking to update or change any fields on the `User`. In contrast, the `update_` methods _are_ looking to change some data on the `User`, so they require mutable access to the `User` instance. 
```

Comment:
Worth mentioning here: the use of getters and setters was artificial, to teach about `&self` and `&mut self` methods. In a real project, instead of writing getters and setters, we'd just make the field public, like this:

```rust
#[derive(Clone, Debug)]
pub struct User {
    pub name: String,
    pub age: u32,
    pub weight: f32,
}
```

Then, users can just use or set `user.name`, `user.age`, `user.weight` wherever they want, without going to the expense of a function call. (Even if the function is inlined, doing so impacts compilation time.)